site/ja/tutorials/images/hub_with_keras.ipynb

```diff
@@ -0,0 +1,1042 @@
+        "id": "IXTB22SpxDLP",
+        "colab": {}
+      },
+      "source": [
+        "plt.figure(figsize=(10,9))\n",
+        "plt.subplots_adjust(hspace=0.5)\n",
+        "for n in range(30):\n",
+        "  plt.subplot(6,5,n+1)\n",
+        "  plt.imshow(image_batch[n])\n",
+        "  plt.title(predicted_class_names[n])\n",
+        "  plt.axis('off')\n",
+        "_ = plt.suptitle(\"ImageNet predictions\")"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "FUa3YkvhxDLM"
+      },
+      "source": [
+        "画像の属性については、「 LICENSE.txt 」ファイルを参照してください。\n",
+        "\n",
+        "結果は完全には程遠いですが、これらがこのモデルのためにトレーニングされたクラスでない事を考えると悪くはないです（ 「デイジー」を除く）。\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "JzV457OXreQP"
+      },
+      "source": [
+        "### ヘッドレスモデルをダウンロードする\n",
+        "\n",
+        "TensorFlow Hub はさらに、最上位の分類層がないモデルを配布することもしています。これらは転移学習を簡単に行うために使用することができます。\n",
+        "\n",
+        "tfhub.dev からの[TensorFlow 1.x の画像特徴ベクトルの URL](https://tfhub.dev/s?module-type=image-feature-vector) は全てここで動作します。"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "cellView": "both",
+        "colab_type": "code",
+        "id": "4bw8Jf94DSnP",
+        "colab": {}
+      },
+      "source": [
+        "feature_extractor_url = \"https://tfhub.dev/google/tf2-preview/mobilenet_v2/feature_vector/2\" #@param {type:\"string\"}"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "sgwmHugQF-PD"
+      },
+      "source": [
+        "モジュールを作成し、予想される画像サイズを確認します。"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "5wB030nezBwI",
+        "colab": {}
+      },
+      "source": [
+        "feature_extractor_layer = hub.KerasLayer(feature_extractor_url,\n",
+        "                                         input_shape=(224,224,3))"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "GUY-5Eyzuzlu"
+      },
+      "source": [
+        "特徴抽出器は、各画像に対して1280要素のベクトルを返します。"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "Of7i-35F09ls",
+        "colab": {}
+      },
+      "source": [
+        "feature_batch = feature_extractor_layer(image_batch)\n",
+        "print(feature_batch.shape)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "CtFmF7A5E4tk"
+      },
+      "source": [
+        "トレーニングが新しい分類層のみを修正するように、特徴抽出層の変数を凍結します。"
```

Comment:
[Weak Suggestion]
I think the below are better as a translation of `freeze`:
- 固定する
- フリーズする (not translate because "freeze" is a technical term of TensorFlow)
- freeze する